src/Console/Command/ShowCommand.php

```diff
@@ -0,0 +1,507 @@
+            ];
+
+            if (!$this->hideConfigured) {
+                $row['is_configured'] = $fixer['is_configured'] ? sprintf('<fg=green;>%s</>', self::THICK) : sprintf('<fg=red;>%s</>', self::CROSS);
+            }
+
+            if (!$this->hideEnabled) {
+                $row['is_enabled'] = $fixer['is_enabled'] ? sprintf('<fg=green;>%s</>', self::THICK) : sprintf('<fg=red;>%s</>', self::CROSS);
+            }
+
+            if (!$this->hideRisky) {
+                $row['is_risky'] = $fixer['is_risky'] ? sprintf('<fg=green;>%s</>', self::THICK) : sprintf('<fg=red;>%s</>', self::CROSS);
+            }
+
+            if (!$this->hideInherited) {
+                $row['is_inherited'] = $fixer['is_inherited'] ? sprintf('<fg=green;>%s</>', self::THICK) : sprintf('<fg=red;>%s</>', self::CROSS);
+            }
+
+            if (!$this->hideDeprecated) {
+                $row['is_deprecated'] = $fixer['is_deprecated'] ? sprintf('<fg=green;>%s</>', self::THICK) : sprintf('<fg=red;>%s</>', self::CROSS);
+            }
+
+            if (!$this->hideCustom) {
+                $row['is_custom'] = $fixer['is_custom'] ? sprintf('<fg=green;>%s</>', self::THICK) : sprintf('<fg=red;>%s</>', self::CROSS);
+            }
+
+            return $row;
+        }, $rows);
+    }
+
+    private function calculateComparison()
+    {
+        $this->undefinedFixers = array_diff_key($this->fixerList, $this->configuredFixers, $this->enabledFixers);
+    }
+
+    /**
+     * @param OutputInterface $output
+     */
+    private function dumpComparison(OutputInterface $output)
+    {
+        if (empty($this->undefinedFixers)) {
+            $output->writeln("\nYou are aware of all exsisting rules! Yeah!");
+
+            return;
+        }
+
+        $line = var_export(
+            array_map(function () {return false; }, $this->undefinedFixers),
+            true
+        );
+
+        $output->writeln(
+            sprintf(
+            "\nCopy and paste the following <info>%d</info> undefined rules in your <comment>%s</comment> config file %s.\n\n"
+            ."// Below the rules I don't want to use\n"
+            ."%s;\n"
+            .'// END Rules to never use',
+            \count($this->undefinedFixers),
+            $this->configName,
+            $this->configFile,
+            $line
+        )
+        );
+    }
+
+    /**
+     * @param FixerInterface $fixer
+     *
+     * @return bool
+     */
+    private function isFixerConfigured(FixerInterface $fixer)
+    {
+        return isset($this->configuredFixers[$fixer->getName()]);
+    }
+
+    /**
+     * @param FixerInterface $fixer
+     *
+     * @return bool
+     */
+    private function isFixerEnabled(FixerInterface $fixer)
+    {
+        return isset($this->enabledFixers[$fixer->getName()]);
+    }
+
+    /**
+     * @param FixerInterface $fixer
+     *
+     * @return bool
+     */
+    private function isFixerEnabledTroughInheritance(FixerInterface $fixer)
```

Comment:
Need to correct all `Trough` => `Through`

Reply:
Done